src/Whoops/Resources/views/header.html.php

```diff
@@ -15,6 +15,25 @@
   <div class="exc-message">
     <?php if (!empty($message)): ?>
       <span><?php echo $tpl->escape($message) ?></span>
+
+
+      <?php if (count($previousMessages)): ?>
+        <div class="exc-title prev-exc-title">
+          <span class="exc-title-secondary">Previous Exceptions</span>
+        </div>
+
+        <ul>
+          <?php foreach ($previousMessages as $i => $previousMessage): ?>
+            <li>
+              <span class="prev-exc-code">(<?php echo $previousCodes[$i] ?>)</span>
```

Comment:
In the top-level exception we used to have the exc.-behind the message

Reply:
Oops, your're right. :)
![image](https://user-images.githubusercontent.com/1024005/47255938-311dca00-d478-11e8-980d-cf3ee75c79b7.png)
Changed now